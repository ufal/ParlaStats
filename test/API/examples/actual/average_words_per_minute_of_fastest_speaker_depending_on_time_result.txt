[{'speech.earliest_timestamp': '12:48:00', 'average_wpm': 161.34031413612567}]

Running time 9.771728515625 ms

+---------------------------+--------------------+
| speech.earliest_timestamp |    average_wpm     |
+---------------------------+--------------------+
|          12:48:00         | 161.34031413612567 |
+---------------------------+--------------------+
