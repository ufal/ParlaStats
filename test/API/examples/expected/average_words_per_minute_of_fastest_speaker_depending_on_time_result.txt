[{'speech.earliest_timestamp': '12:48:00', 'average_wpm': 161.34031413612567}]

Running time 10.3564453125 ms

+---------------------------+--------------------+
| speech.earliest_timestamp |    average_wpm     |
+---------------------------+--------------------+
|          12:48:00         | 161.34031413612567 |
+---------------------------+--------------------+
